core/themes/bartik/css/style.css

```diff
@@ -1185,6 +1185,32 @@ ul.action-links li a {
   padding: 0;
 }
 
+ /* -------------- Hero blocks  -------------- */
+
+.hero-block {
+  position: relative;
+  padding: 8% 4%; /* All padding shrinks/grows according to available width. */
+  min-height: 128px;
```

Comment:
I was thinking of recommending vh here, but now I'm not so sure.

I did see @jenlampton mentioned that the 'clunkiness' of the breakpoints wasn't great, we could add:
```
  -webkit-transition: min-height 0.25s;
  transition: min-height 0.25s;
```